zabbixsender/zabbixsvc/zabbixsvc.go

```diff
@@ -38,25 +42,58 @@ type ZabbixResponse struct {
 
 //JSONHandler handles alerts
 type JSONHandler struct {
-	Sender      *zabbixsnd.Sender
-	KeyPrefix   string
-	DefaultHost string
+	Sender       *zabbixsnd.Sender
+	KeyPrefix    string
+	DefaultHost  string
+	DefaultHosts DefaultHosts
 	//TODO: Introduce annotation to host mapping?
 }
 
+//TODO change to specific hosts and values
+type DefaultHosts struct {
```

Comment:
Consider just using `map[string]string` instead of `DefaultHosts`,  as the data is `receiverName`:`hostName`